optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SingleBenchmarkResult.java

```diff
@@ -201,6 +210,10 @@ public boolean isSuccess() {
         return succeeded != null && succeeded.booleanValue();
     }
 
+    public boolean isInitialized() {
+        return initialized != null && initialized.booleanValue();
```

Comment:
Hmm, but I do it myself in isSuccess() and isFailure(), because the solver config can throw an exception at which point it never gets to the success/failure/initialized population code, but it does report it in the report. This is because the benchmarker is the only thing in all of planner that is special in the sense that it doesn't fail fast: if a single benchmark report fails, the benchmarker takes note of that and continues with the other benchmarks: it does write the report, but it does also throw an exception, at the very last moment. This is as special case, "an exception that confirms the rule".

So my previous comment doesn't apply here :) Ignore it.